proto/query.proto

```diff
@@ -49,6 +49,10 @@ message QueryRequest {
   string sparql = 1;
 }
 
+message SparqlRequest {
```

Comment:
Curious why you didn't want to reuse `QueryRequest`.  If that's going to be deprecated, add a comment/TODO?

Reply:
That one has a different param `sparql` inside. Want to avoid confusing there.